vendor-extra/JatsContentWorkflowBundle/tests/Workflow/MoveJatsAssetsTest.php

```diff
@@ -334,6 +338,84 @@ public function it_checks_the_origin_uri() : void
         $this->assertXmlStringEqualsXmlString($expected, $task->getDocument());
     }
 
+    /**
+     * @test
+     */
+    public function it_fails_if_the_asset_can_not_be_loaded() : void
+    {
+        $mover = new MoveJatsAssets('.+', 'http://public-assets/path', $this->filesystem, $this->client);
+
+        $document = FluentDOM::load(
```

Comment:
any chance to extract `$document` and `$mock`'s expectations if/when they are repeated throughout the tests?

Reply:
Though there is always some variation like the response code or a minor path.